broker-node/app/Console/Kernel.php

```diff
@@ -28,9 +28,9 @@ protected function schedule(Schedule $schedule)
             ->everyMinute()
             ->sendOutputTo('/var/CHUNKLOG');
 
-        $schedule->command('GetFreshTips:getTips')
-            ->everyFiveMinutes()
-            ->sendOutputTo('/var/TIPSLOG');
+//        $schedule->command('GetFreshTips:getTips')
```

Comment:
Did you mean to keep this commented out?

Reply:
Yeah.  I will double check later tonight, but I think the tip selection service is doing more harm than good.